### ISSUE TITLE
Do you think blockchain technology is really useful? why?

### ISSUE THREAD

**Issue by Sugaritem-GitHub** [Sugaritem/Operation#7]
Do you think blockchain technology is really useful? why?

**Comment by JiaMiShiDai** [comment]
sure.Blockchain is the best solution to solve the trust problem in the digital age

**Comment by Qiqutimes** [comment]
Of course, blockchains can help solve many issues in our real life, for example, IoT period,  banking period and so on.

**Comment by ccc888** [comment]
It's very useful. The end of blockchain can solve the trust problem.
